### PR TITLE
[Maintenance] Refactor services' ids according to the new convention

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
| --------------- | -----
| Branch?         | 2.0
| Bug fix?        | no
| New feature?    | no
| Related tickets | #315

### CODE REVIEW

File: config/services.xml [hunk 1]
```diff
@@ -263,48 +251,45 @@
...
+        <service id="Sylius\PayPalPlugin\Provider\PaymentReferenceNumberProviderInterface" alias="sylius_paypal.provider.payment_reference_number" />
 
-        <service id="Sylius\PayPalPlugin\Command\CompletePaidPaymentsCommand">
+        <service id="sylius_paypal.console.command.complete_paid_payments" class="Sylius\PayPalPlugin\Console\Command\CompletePaidPaymentsCommand">
             <argument type="service" id="sylius.repository.payment" />
             <argument type="service" id="sylius.manager.payment" />
             <argument type="service" id="sylius_paypal.api.cache_authorize_client" />
             <argument type="service" id="sylius_paypal.api.order_details" />
             <argument type="service" id="sylius_abstraction.state_machine" />
             <tag name="console.command" command="sylius:pay-pal-plugin:complete-payments" />
```
Comment (GSadee) [inline config/services.xml]: Not needed
```suggestion
            <tag name="console.command" />
```

File: src/Console/Command/CompletePaidPaymentsCommand.php [hunk 2]
```diff
@@ -37,21 +35,9 @@ public function __construct(
...
-                sprintf(
-                    'Passing an instance of "%s" as the fifth argument is deprecated and will be prohibited in 2.0. Use "%s" instead.',
-                    FactoryInterface::class,
-                    StateMachineInterface::class,
-                ),
-            );
-        }
     }
 
     protected function configure(): void
```
Comment (GSadee) [inline src/Console/Command/CompletePaidPaymentsCommand.php]: Replace this configuration method with the attribute

Comment (GSadee) [reply src/Console/Command/CompletePaidPaymentsCommand.php]: I also wonder whether we should not rename this command
```
sylius:paypal:complete-payments
sylius:paypal-plugin:complete-payments
sylius-paypal:complete-payments
```
🤔

Comment (Wojdylak) [reply src/Console/Command/CompletePaidPaymentsCommand.php]: To be consistent with the `InvoicingPlugin`, I chose `sylius-paypal:complete-payments`.

**Review by GSadee — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Pull request merged**